downloads/swan-lake-release-notes/swan-lake-2201.7.0/RELEASE_NOTE.md

```diff
@@ -62,6 +81,30 @@ public function main() {
 
 ### Improvements
 
+#### Improved resolving logic of module level constructs.
+
+With this new feature,
+
+- Made the type node in constant declarations optional. Both of the following cases are valid.
+
+    ```ballerina
+    const map<string> PERSONA = {firstName: "Emily", lastName: "Johnson"};
+    const PERSONB = {firstName: "David", lastName: "Thompson"};
```

Comment:
Again, may make sense to have something like an error data structure rather than person.

Reply:
error type is not yet supported for constants. Are you referring to something like this `const HTTPDTAILS = {httpCode: 200, message: "OK"};`

Reply:
Yeah, maybe calling it something like `HTTP_OK`.